Source/WebCore/css/CSSFontFaceSrcValue.cpp

```diff
@@ -88,8 +88,10 @@ Ref<CSSFontFaceSrcResourceValue> CSSFontFaceSrcResourceValue::create(ResolvedURL
 
 std::unique_ptr<FontLoadRequest> CSSFontFaceSrcResourceValue::fontLoadRequest(ScriptExecutionContext& context, bool isInitiatingElementInUserAgentShadowTree)
 {
-    if (m_cachedFont)
-        return makeUnique<CachedFontLoadRequest>(*m_cachedFont);
+    if (m_cachedFont) {
+        RefPtr document = dynamicDowncast<Document>(&context);
```

Comment:
No need, we can just pass the context, this way the logging will work in workers too.

Reply:
hmm I passed `context` instead of `&context`. Trying to recompile with `&context`

Reply:
ah no. Everything is as you asked. :) confused by my own changes.